### PR TITLE
Initial wallet app plus some encryption of security image

### PR DESCRIPTION
**Pull request by davidkelly**
This is @zfox23's new wallet UI, with my additions to allow us to serve up an encrypted security image.  For now, the image is encrypted as hifi_image (using the salt which is hardcoded for now), and placed next to the key file (hifikey) in AppData/Roaming/High Fidelity/Interface.  We will consolidate into one shortly.

### CODE REVIEW

File: unpublishedScripts/marketplace/spectator-camera/spectatorCamera.js [hunk 1]
```diff
@@ -153,7 +153,7 @@
     //   -showSpectatorInDesktop: Set to "true" to show the "SPECTATOR" app in desktop mode.
     var button = false;
     var buttonName = "SPECTATOR";
-    var showSpectatorInDesktop = false;
+    var showSpectatorInDesktop = true;
```
Comment (zfox23) [inline unpublishedScripts/marketplace/spectator-camera/spectatorCamera.js]: Heh. Whoops. This shouldn't be in here.

Comment (davidkelly) [inline unpublishedScripts/marketplace/spectator-camera/spectatorCamera.js]: ohhh wow I _just_ saw this too

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2698/HighFidelity-Beta-PR11235-57d973851e559cb7da99edafdd46afde7446f8e2.dmg

**Comment by marko8904** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2705/HighFidelity-Beta-PR11235-57d973851e559cb7da99edafdd46afde7446f8e2.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2705/HighFidelity-Beta-PR11235-57d973851e559cb7da99edafdd46afde7446f8e2.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2715/HighFidelity-Beta-PR11235-c77dafdf129dcc5bb87864ac5b7998a5fc7b5523.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2715/HighFidelity-Beta-PR11235-c77dafdf129dcc5bb87864ac5b7998a5fc7b5523.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2721/HighFidelity-Beta-PR11235-d398adc1950f3c0b80c100fd0aaab03f62303839.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2726/HighFidelity-Beta-PR11235-698644c3b43d17260de8dd445c2ae912cb93423d.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2733/HighFidelity-Beta-PR11235-a299aa7e2407bfbef5c9953b586dcebf67702623.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2733/HighFidelity-Beta-PR11235-a299aa7e2407bfbef5c9953b586dcebf67702623.dmg

**Comment by howard-stearns** [comment]
lots of bugs, but I've recorded them in our trello board, along with changes (for later) to the code, and with the idea of developing a test plan.
So I'll label this "approved" as our stake-in-the-ground against which to fix things.

**Pull request merged**